frontend/fingertips-frontend/components/charts/SpineChart/helpers/healthDataRequestAreas.test.ts

```diff
@@ -0,0 +1,108 @@
+import { areaCodeForEngland } from '@/lib/chartHelpers/constants';
+import { SearchParams } from '@/lib/searchStateManager';
+import { healthDataRequestAreas } from './healthDataRequestAreas';
+import { englandAreaType } from '@/lib/areaFilterHelpers/areaType';
+import { Area } from '@/generated-sources/ft-api-client';
+
+const mockArea1 = { code: 'A1', name: 'Area 1' } as Area;
+
+describe('healthDataRequestAreas', () => {
+  it('returns only selected area codes if England is included', () => {
+    const searchState = {
+      [SearchParams.AreasSelected]: [areaCodeForEngland],
+      [SearchParams.AreaTypeSelected]: 'LocalAuthority',
+    };
+
+    const result = healthDataRequestAreas(searchState, [mockArea1]);
+
+    expect(result).toEqual([
+      {
+        areaCodes: [areaCodeForEngland],
```

Comment:
would it make sense as part of this test to include more selected areas since we are checking only the selectedAreas are returned if england is included.